bridgetown.config.yml

```diff
@@ -24,6 +24,10 @@ template_engine: erb
 
 include: ["_redirects"]
 
+metadata:
```

Comment:
💅 I don't think it was necessary to put this here, it's more for stuff that needs to be changed by editors :)